packages/overlay/src/integrations/sentry/components/insights/agents/AITraceDetails.tsx

```diff
@@ -0,0 +1,326 @@
+import { useState } from 'react';
```

Comment:
Big file ahead. Once we add support for more AI libraries, we’ll better understand the common components in this detail panel. That should help us break it down